x-pack/plugins/data_enhanced/public/index.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { PluginInitializer } from 'kibana/public';
+import {
+  DataEnhancedPublicPlugin as Plugin,
```

Comment:
No need to add `Public` to everything anymore
`DataEnhancedPlugin` is the way to go.

FYI https://github.com/elastic/kibana/pull/55281 was merged, generating all the right shapes, for refernce.